public/app/features/alerting/unified/components/rules/RuleActionsButtons.tsx

```diff
@@ -178,10 +181,22 @@ export const RuleActionsButtons = ({ rule, rulesSource }: Props) => {
         />
       );
     }
+  }
 
-    if (canDeleteRule) {
-      moreActions.push(<Menu.Item label="Delete" icon="trash-alt" onClick={() => setRuleToDelete(rule)} />);
-    }
+  if (ruleExtensionLinks.length > 0) {
+    moreActions.push(
+      <Menu.Divider />,
+      ...ruleExtensionLinks.map((extension) => (
+        <Menu.Item key={extension.id} label={extension.title} icon={extension.icon} onClick={extension.onClick} />
+      ))
+    );
+  }
+
+  if (rulerRule && canDeleteRule) {
```

Comment:
Yeah, I see why we need to coordinate what we're doing between this and https://github.com/grafana/grafana/pull/86071 😁